jest.setup.js

```diff
@@ -1,12 +1,38 @@
 import 'react-native-gesture-handler/jestSetup';
 import mockNetInfo from '@react-native-community/netinfo/jest/netinfo-mock.js';
+import mockSafeAreaContext from 'react-native-safe-area-context/jest/mock';
+import {NativeModules} from 'react-native';
 import {setUpTests} from 'react-native-reanimated';
 import {jest} from '@jest/globals';
 
 setUpTests();
 
+const mockExpoLocation = {
+  Accuracy: {
+    Lowest: 1,
+    Low: 2,
+    Balanced: 3,
+    High: 4,
+    Highest: 5,
+    BestForNavigation: 6,
+  },
+  getForegroundPermissionsAsync: jest.fn(() =>
+    Promise.resolve({
+      status: 'granted',
+      expires: 'never',
+      granted: true,
+      canAskAgain: true,
+    }),
+  ),
+  getLastKnownPositionAsync: jest.fn(() => Promise.resolve(null)),
+  watchPositionAsync: jest.fn(() => Promise.resolve({remove: jest.fn()})),
+};
+jest.mock('expo-location', () => mockExpoLocation);
```

Comment:
soft recommendation to move this to `__mocks__/expo-location.ts`

Reply:
Ah, didn't know we already had one of those. Moved in d164b79225803ee98c03ecc170215eabc01ba85d.